components/clubs.jsx

```diff
@@ -0,0 +1,164 @@
+var React = require('react');
```

Comment:
Is this file meant to be part of the PR? It seems like a duplicate of `clubs-page.jsx`...